docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/01 - Identifiers.md

```diff
@@ -52,14 +52,17 @@ PSCIDs can be created for new subjects in one of 3 ways: *sequentially generated
      <generation>random</generation> 
      <structure>
          <seq type="static">PROJ</seq>
-         <seq type="siteAbbrev"/>
+         <seq type="projectAbbrev"/>
          <seq type="numeric" length="4" min="1" max="9999"/> <!-- Ex: PROJMTL3994-->
```

Comment:
> Ex: PROJMTL3994

This is a bit of a confusing example. Would the siteAbbrev in this case be `PROJMTL` or is this using both aliases?

It may be good to clarify this to be something like

> Ex: PROJMTL3994 given a Site with Alias $X and a Project with Alias $Y

Reply:
I just forgot to change the example after going from site to project. but ur suggestion is a good idea, I'll add it.

Side note, would it work here ? are we not limited to 1 prefix ?

Reply:
I'm pretty sure only one is possible, yeah.

Reply:
@johnsaigle take a look again ?